playground/backend/internal/code_processing/code_processing.go

```diff
@@ -95,7 +95,7 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	// Prepare
 	logger.Infof("%s: Prepare() ...\n", pipelineId)
 	prepareFunc := executor.Prepare()
-	go prepareFunc(successChannel, errorChannel)
+	go prepareFunc(successChannel, errorChannel, &validationResults, lc.GetAbsoluteSourceFilePath())
```

Comment:
Why do you need GetAbsoluteSourceFilePath?

Reply:
Done